links/systems.json

```diff
@@ -0,0 +1,180 @@
+	"URL":         "https://duckumentation.datasektionen.se",
+	"Icon":        "https://duckumentation.datasektionen.se/images/icons/favicon-96x96.png",
+	"Color":       "#4caf50"
+}, {
+	"Name":        "DUrn",
+	"Description": "Röstningssystem vid urnval",
+	"URL":         "https://durn.datasektionen.se",
+	"Icon":        "https://durn.datasektionen.se/public/favicon.png",
+	"Color":       "#0033cc"
+}, {
+	"Name":        "GitHub",
+	"Description": "Datasektionens GitHub",
+	"URL":         "https://github.com/datasektionen",
+	"Icon":        "https://github.com/favicon.ico",
+	"Color":       "#000"
+}, {
+	"Name":        "Metaspexet",
+	"Description": "Datas och Medias spex!",
+	"URL":         "https://metaspexet.se",
+	"Icon":        "https://metaspexet.se/images/favicon.svg",
+	"Color":       "#800000",
+	"IconStyle":   "background-color: #6f1d1b; border: 2px solid #6f1d1b; border-radius: 50%"
+}, {
+	"Name":        "META-TV",
+	"Description": "Nå ut till hundratals genom META-TV",
+	"URL":         "https://tv.datasektionen.se",
+	"Icon":        "https://tv.datasektionen.se/favicon.png",
+	"Color":       "#009688"
+}, {
+	"Name":        "Nyhetssystemet",
+	"Description": "Skriv och posta nyheter till hemsidan",
+	"URL":         "https://calypso.datasektionen.se",
+	"Icon":        "https://calypso.datasektionen.se/favicon-96x96.png",
+	"Color":       "#ec407a"
+}, {
+	"Name":        "Pico",
+	"Description": "Länkförkortare",
+	"URL":         "https://dsekt.se",
+	"Icon":        "https://dsekt.se/favicon-96x96.png",
+	"Color":       "#03a9f4"
+}, {
+	"Name":        "Pls",
+	"Description": "Lär dig ta ett nej",
+	"URL":         "https://pls.datasektionen.se",
+	"Icon":        "https://pls.datasektionen.se/images/logos/favicon-96x96.png",
+	"Color":       "#9c27b0"
+}, {
+	"Name":        "Sektionswikin",
+	"Description": "Här hittar du massvis ackumulerad kunskap.",
+	"URL":         "https://wiki.datasektionen.se",
+	"Icon":        "https://wiki.datasektionen.se/resources/assets/skold_svart_thumb.png",
+	"Color":       "#ff9800"
+}, {
+	"Name":        "Status",
+	"Description": "Ta reda på om något av våra system är nere",
+	"URL":         "https://status.datasektionen.se",
+	"Icon":        "https://userfiles.uptimerobot.com/img/623458-1649958820-icon.png",
+	"Color":       "#32cd32"
+}, {
+	"Name":        "STÖn",
+	"Description": "Suck",
+	"URL":         "https://ston.datasektionen.se",
+	"Icon":        "https://ston.datasektionen.se/images/logos/favicon-96x96.png",
+	"Color":       "#ec5f99",
+	"Sensitive":   true
+}, {
+	"Name":        "Yoggi",
+	"Description": "Ladda upp statiska filer i sektionssyfte",
+	"URL":         "https://static.datasektionen.se",
+	"Icon":        "https://static.datasektionen.se/favicon-96x96.png"
+}, {
+	"Name":        "METAcraft",
+	"Description": "Datas och Medias Minecraftserver",
+	"URL":         "https://metacraft.nu",
+	"Icon":        "https://metacraft.nu/logo-square.png"
+}
+]
+
```

Comment:
```suggestion
```

Reply:
> POSIX defines a text file as a file that contains characters organized into zero or more lines, where lines are sequences of zero or more non-newline characters plus a terminating newline character, normally LF.
Thanks for reminding me to add an extra newline to `sm.json` as well!

Reply:
Nvm